services/iam/drivers/IamAccount.py

```diff
@@ -246,13 +257,8 @@
                 return 0
 
     def _checkHasOrganization(self):
-        try:
-            resp = self.orgClient.describe_organization()
-        except botocore.exceptions.ClientError as e:
-            ecode = e.response['Error']['Code']
-            if ecode == 'AWSOrganizationsNotInUseException':
+        if (self.organizationIsEnabled == False):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)